File: Iniciante/Pagina 04/1145/1145.c
#include <stdio.h>
 
int main() {
    int i, j, total, sizelinha, numlinha, cont = 1;
    
    scanf("%d %d", &sizelinha, &total);
    
    numlinha = total / sizelinha;

    for (i = 0; i < numlinha; i++) {
        for (j = 0; j < sizelinha; j++) {
            if (j == sizelinha - 1){
                printf("%d\n", cont);
                cont++; 
            } else {
                printf("%d ", cont);
                cont++; 
            }
        }
    }
    
    return 0;
}

File: Iniciante/Pagina 04/1145/1145.py
sizelinha, total = map(int, input().split())
i = 0
j = 0
cont = 1
numlinha = total // sizelinha

for i in range(numlinha):
    for j in range(sizelinha):
        if j == sizelinha - 1:
            print(cont)
            cont += 1
        else:
            print(cont, end=' ')
            cont += 1
 